### ISSUE TITLE
Upgrading from 0.12 to 0.13.3 changes behaviour of property access on booleans

### ISSUE THREAD

**Issue by ruudk** [overblog/GraphQLBundle#724]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? |no
| BC Break report? | yes
| RFC?             |no
| Version/Branch   | v0.13.3

When we upgraded our large GraphQL API we noticed that some boolean fields returned null instead of booleans. Upon investigating we found out that some booleans need a resolve attribute like this:
```diff
 valid:
   type: 'Boolean!'
+  resolve: '@=value.isValid()'
```

I think this is related to the removal of the Symfony PropertyAccess component. That component would not only search for `get{$fieldName}` but also for `is{$fieldName}`. 

Can this be added back?

**Comment by mcg-web** [comment]
This will be fix by #720.

**Comment by ruudk** [comment]
@mcg-web Awesome, I'll close this one and wait for the next release. Thanks for the fast reply ⚡
